common/network-types/src/types.rs

```diff
@@ -1,4 +1,6 @@
 use crate::errors::NetworkTypeError;
+use hickory_resolver::name_server::ConnectionProvider;
```

Comment:
Pls add hickory_resolver and a proper log level to the configuration for tests and docker-compose setup. Note that it produces multiline logs.

Reply:
Does it? I haven't seen it in the logs yet, but will check it out.